pyproject.toml

```diff
@@ -39,6 +39,7 @@ select = [
   "UP032",
   "UP034",
   "T20",
+  "TID252",
```

Comment:
ref: https://beta.ruff.rs/docs/rules/relative-imports/

Reply:
Ruff provides an option to only ban beyond-parent relative imports: 
https://beta.ruff.rs/docs/settings/#flake8-tidy-imports-ban-relative-imports

Reply:
Agreed, let's just ban all relative imports